src/sbvr-api/sbvr-utils.ts

```diff
@@ -1020,11 +1032,37 @@ export class PinejsClient extends PinejsClientCore {
 	}) {
 		return (await runURI(method, url, body, tx, req, custom)) as object;
 	}
+
+	public post<TResource extends keyof M & string>(
+		params: {
+			resource: TResource;
+			options: Params<M[TResource]>['options'] & { returnResource: false };
```

Comment:
We need the overrides in order to properly type this `returnResource: false` case. I'm thinking maybe move it into pinejs-client but if doing so I'd want to properly type the primary key (rather than the `id` hardcoding)